cmd/admin-kms-key-status.go

```diff
@@ -63,29 +68,48 @@ func mainAdminKMSKeyStatus(ctx *cli.Context) error {
 	status, e := client.GetKeyStatus(globalContext, keyID)
 	fatalIf(probe.NewError(e), "Failed to get status information")
 
-	printMsg(kmsKeyStatusMsg(*status))
+	printMsg(kmsKeyStatusMsg{
+		KeyID:         status.KeyID,
+		Encryption:    status.EncryptionErr == "",
+		Decryption:    status.DecryptionErr == "",
+		EncryptionErr: status.EncryptionErr,
+		DecryptionErr: status.DecryptionErr,
+	})
 	return nil
 }
 
-type kmsKeyStatusMsg madmin.KMSKeyStatus
+type kmsKeyStatusMsg struct {
+	KeyID         string `json:"keyId"`
+	Encryption    bool   `json:"encryption"`
```

Comment:
This is redundant state. `Encryption` is always `false` if `EncryptionErr != ""` resp. `true` if `EncryptionErr == ""`. Also a boolean doesn't say much - "What does `Encryption == true` means?"

Reply:
It is on purpose to make sure that json output is consistent it is okay to have such redundant state